pom.xml

```diff
@@ -1134,6 +1135,7 @@
                 <module>spring-core-6</module>
                 <module>data-structures</module>
                 <module>ddd-contexts</module>
+                <module>jackson-modules</module>
```

Comment:
we will have to remove it from default-first and integration-lite-first also

Reply:
Ok, done